support-frontend/assets/pages/supporter-plus-thank-you/supporterPlusThankYou.tsx

```diff
@@ -55,10 +55,13 @@ export const firstColumnContainer = css`
 
 export const headerContainer = css`
 	${from.desktop} {
-		width: 60%;
+		width: 70%;
 	}
 	${from.leftCol} {
-		width: calc(50% - ${space[3]}px);
+		width: calc(60% - ${space[3]}px);
+	}
+	${from.wide} {
+		width: calc(55% - ${space[3]}px);
 	}
 `;
```

Comment:
Why were these header widths updated? As the width of the header copy now differs from the figma

Reply:
This change was to get the copy text ('Thankyou for your contribution...') to display on 2 lines only at >desktop  breakpoints to match the supplied Figma design (it will show over 3 lines otherwise, at wide, the url link gets split across lines). That ok or shall I revert?

Reply:
Did you mean 'Thankyou for your supporting us...'? 
The "today" part on certain breakpoints looks like it should be on the second line?

Reply:
Ok, have reverted this now to the original code. We will get single contribution copy in the header over 3lines (starting  'Thankyou for your contribution...') but it will not modify the header line widths (starting 'Thankyou for supporting us today....') which sounds like the priority. Ok to put this through now?

Reply:
Helene says she is not worried about subheading copy being on more lines than the supplied Figma file. So the revert seems appropriate in this case. Will resolve after standup.